### ISSUE TITLE
[Renoise | Linux] ZLEqualizer crashes DAW when adjusting lookahead

### ISSUE THREAD

**Issue by 0CCULTIST** [ZL-Audio/ZLEqualizer#77]
When adjusting lookahead ZLEqualizer is very susceptible to crashing. 

```
CrashLog: Caught signal: SIGSEGV (11) ...
CrashLog: Caught signal: SIGABRT (6) ...

VstPlugs: Instantiate FAILED (caught structured exception) !!!

CrashLog: Caught signal: SIGSEGV (11) ...
CrashLog: /lib/x86_64-linux-gnu/libc.so.6(+0x3c050) [0x7fa7a4369050]
CrashLog: renoise() [0xb312d6]
CrashLog: /lib/x86_64-linux-gnu/libc.so.6(+0x5400a) [0x7fa7a438100a]
CrashLog: [(nil)]
CrashLog: [0x7ffdb28111d0]

Application: Caught an unhandled fatal exception (Thread: GUI)!
Application: Saving a backup...
```

Manually entering a value seems to not cause any issues. I believe the issue stems from the latency being rapidly changed when dragging the slider. Making it update only when you release the mouse button may help with stability.

**Comment by zsliu98** [comment]
It seems that the SIGSEGV is raised from the GUI(message) thread. May I ask whether the crash happens when you automate the `lookahead` para?

**Comment by 0CCULTIST** [comment]
> It seems that the SIGSEGV is raised from the GUI(message) thread. May I ask whether the crash happens when you automate the `lookahead` para?

Apologies for the very late reply. I actually can't reproduce any crashes when messing around with the latency (including modulating it) using the latest version

It maybe that my DAW is more stable now that I've started minimizing the amount of plugins I use via Yabridge/Wine, though I did definitely experience crashes specifically when changing the lookahead on ZLEqualizer. Closing for now until anyone can reproduce.
